core/tools/generate_names.cc

```diff
@@ -277,9 +279,11 @@ NameDef names[] = {
     // on a class to lookup an associated symbol with some extra info.
     {"sealedSubclasses", "sealed_subclasses"},
 
+    {"classMethodsTemp", "class_methods"}, // TODO: When the other PR gets merged this should be named "classMethods"
```

Comment:
By other PR I'm talking about https://github.com/sorbet/sorbet/pull/3424/files#diff-8b857b0dc241f308e4dfaf9cad799beeL282 . I want to name this `"classMethods"` once that is merged.